langchain/src/llms/tests/bedrock.int.test.ts

```diff
@@ -249,7 +278,7 @@ test("Test Bedrock LLM: stream method", async () => {
     console.log(chunk);
     chunks.push(chunk);
   }
-  expect(chunks.length).toBeGreaterThan(1);
+  expect(chunks.length).toBeGreaterThanOrEqual(1);
```

Comment:
Shouldn't this be greater than 1?

Reply:
The mock I added does streaming, but just 1 chunk.  
https://github.com/hwchase17/langchainjs/blob/7f2271769488152259791226d4c3767f60919ae9/langchain/src/llms/tests/bedrock.int.test.ts#L321-L337
That's why I swapped this.